Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16-alpine
```

Comment:
This will make the build time massive. I suggest we do a separate Dockerfile for each app.

Reply:
Only when we have a change in the dependencies, we need to build again. When it builds, it installs all dependencies for web and api and reuse the image for both. If we separate the installation and images, it couldn't make the time shorter than this. Also I'm not sure how we can separate it in the yarn workspaces.

Reply:
How long does it take in your env? it took just a few minutes in my env.

Reply:
https://github.com/bcgov/myhealthBC-symptom-checker/pull/11